### ISSUE TITLE
publish serde 0.7.3

### ISSUE THREAD

**Issue by oli-obk** [serde-rs/serde#300]
This is blocking an [upgrade of serde-xml to 0.7](https://github.com/serde-rs/xml/pull/14)

**Comment by jimmycuadra** [comment]
A new crate release is also necessary to bump the version numbers of aster/quasi/etc. for the latest libsyntax changes. Right now you can't compile on the latest nightly Rust. :(

**Comment by kdy1997** [comment]
This causes problem 

Cargo.lock
```toml
[[package]]
name = "serde_codegen"
version = "0.7.2"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "aster 0.15.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "quasi 0.9.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "quasi_macros 0.9.0 (registry+https://github.com/rust-lang/crates.io-index)",
]
```
Even if I specify 
```toml
quasi_codegen = "^0.10.0"
```
on ```Cargo.toml```,  and I run ```cargo update```, ```Cargo.lock``` contains quasi_macros 0.9.0

But at now, serde is updated to 0.7.4 and it is compiled on nightly.

Environment: c9.io
```
$ cargo --version
cargo 0.11.0-nightly (afac7fc 2016-04-27)

$ rustc --version
rustc 1.10.0-nightly (8da2bcac5 2016-04-28)
```

**Comment by oli-obk** [comment]
sorry, bad timing xD i closed it just when you commented. I'm not sure what the problem is. Can you show your entire `Cargo.toml`?

**Comment by kdy1997** [comment]
I think build failed error (commented above, by @jimmycuadra ) is fixed, but I don't know well about serde_xml.

I tried to test if ```quasi="^0.10.0"``` would work, but at that time, serde is updated to 0.7.4, and build successed.

Cargo.toml:
```toml
serde = { version = "^0.7", optional = true }
serde_json = { version = "^0.7", optional = true }
serde_macros = { version = "^0.7", optional = true }
```

**Comment by kdy1997** [comment]
Sorry for my bad english, but I meaned 'It workes on nightly-2016-04-27'

**Comment by oli-obk** [comment]
ah :) perfect, thanks, closing this as "fixed by using serde 0.7.4"
